redislock.go

```diff
@@ -265,11 +266,12 @@ func ExponentialBackoff(min, max time.Duration) RetryStrategy {
 }
 
 func (r *exponentialBackoff) NextBackoff() time.Duration {
-	r.cnt++
+	atomic.AddUint32(&r.cnt, 1)
 
 	ms := 2 << 25
-	if r.cnt < 25 {
-		ms = 2 << r.cnt
+	cnt := atomic.LoadUint32(&r.cnt)
```

Comment:
instead of reading it back, just use the `cnt` returned from the atomic increment above

Reply:
Updated.